src/bot.ts

```diff
@@ -53,6 +55,21 @@ const handleCommand = async (message: Discord.Message, command: string, args: st
   }
 };
 
+const handleAnonCommand = async (message: Discord.Message, command: string, args: string[]) => {
+  // log command and its author info
+  logger.info({
+    event: 'command',
+    messageId: message.id,
+    command,
+    args
+  });
+
+  switch (command) {
+    case 'suggest':
+      suggestCmd(message, args);
```

Comment:
i think we can move this back to the normal handle command function since we are logging everything